components/site/layout.tsx

```diff
@@ -37,17 +52,12 @@ const Layout = (props: LayoutProps): JSX.Element => {
           >
             Skip to main content
           </StyledSkipToContent>
-          <Header />
+          <Header getActivatorProps={getActivatorProps} />
           <Box display="flex" width="100%" bg="white" border="400">
-            <Box flex="0">{navigationComponent}</Box>
-            <Box
-              py="500"
-              flex="1"
-              id="main-content"
-              borderLeft="400"
-              overflow="hidden"
-              position="relative"
-            >
+            <Box flex="0" borderRight="400" display={['none', null, 'block']}>
+              {navigationComponent}
+            </Box>
+            <Box p="500" flex="1" id="main-content" overflow="hidden" position="relative">
```

Comment:
`p="500"` needs to be `py="500"` here to allow the full width <hr> in the content